tests/chaos_test.py

```
@@ -1,3 +1,31 @@
+
+    async def fail():
+        raise RuntimeError("boom")
+
+    async def succeed():
+        return 42
+
+    with pytest.raises(RuntimeError):
+        await cb.call(fail)
+    with pytest.raises(RuntimeError):
+        await cb.call(fail)
+    with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)